### PR TITLE
Added the Groupassigner

### PR DESCRIPTION
**Pull request by derrobin154**
## Languages (lang.de.php) (Needs a Translate in other Languages)
Add: 
- $lang["navbar"]["groupassigner"]
- $lang["grouppage"]["title"]
- $lang["grouppage"]["tsuid"]
- $lang["grouppage"]["tsgroup"]
- $lang["grouppage"]["send"],
- $lang["grouppage"]["error"]["offline"]
- $lang["grouppage"]["error"]["notallowed"]
- $lang["grouppage"]["error"]["duplicate"]
- $lang["grouppage"]["success"]["success"]

## Config (config.template.php) (Needs a Change in your Config) 
Add:  
- $config["groupassigner"]
- $config["groups"]["allowgroups"]

## Header (header.php)
When the Groupassigner is active, then "Server Viewer" is a Dropdown Menu with the Link for the Groupassigner, when is inactive then is only a Link to the "Server Viewer"

## Groupassigner (groupassigner.php)
When the Groupassigner is active, then shows a Form with the UID and the ServerGroups was in the Config ($config["groups"]["allowgroups"]).
If the form is manipulated, and the server group is not in $config["groups"]["allowgroups"] then it will also output an error if the server is offline, or the login data for Server Query is incorrect.

### CODE REVIEW

**Comment by toster234** [comment]
Not for use, did you test it?

**Comment by derrobin154** [comment]
I tested it myself, yes.
Is now not as comprehensive as other Groupassigner but fulfils its purpose of assigning a group to a user.
And that will certainly be enough for TS-Website 2.0.

**Review by Bluscream — APPROVED** [review]
Looks about right for now.

Can you add a few more translations though?

**Comment by derrobin154** [comment]
Ouh.. eh.. yes i can create a Translation for English.

File: groupassigner.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                ?>
+            </select>
+            <br><br>
+            <button type="submit" name="absenden" class="btn btn-success " style="margin-left: 45%;"><?php tl($lang["grouppage"]["send"]); ?></button>
+        </form>
+
+
+    </div>
+</div>
+<?php }?>
```
Comment (toster234) [inline groupassigner.php]: ??? 🥇 💯

File: groupassigner.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                $error = "duplicate";
+            }
+
+        }
+    }else{
+         $error = "notallowed";
+    }
+}
+?>
+<?php if($config["groupassigner"] == 1){ ?>
```
Comment (alexandre433) [inline groupassigner.php]: No need to close then open again, just remove 
`?> <?php`

File: groupassigner.php (hunk 1 above)
Comment (alexandre433) [reply groupassigner.php]: line 47 closing tag

**Review by alexandre433 — CHANGES_REQUESTED** [review]

File: groupassigner.php (hunk 1 above)
Comment (alexandre433) [reply groupassigner.php]: but still, if you're going to continue to use php scripting you can just remove the `?> and the <?php` (line 86)
plus you don't need to put a closing tag on the end of a php file.

Comment (derrobin154) [reply groupassigner.php]: I edited it :-)

File: groupassigner.php (hunk 2 above)
Comment (derrobin154) [reply groupassigner.php]: I edited it :-)

**Comment by derrobin154** [comment]
I have added the lang.en.php to the Commit :-)
@Bluscream

**Review by alexandre433 — APPROVED** [review]

**Comment by alexandre433** [comment]
Why close?

**Comment by toster234** [comment]
> Why close?

I upload my version.

**Pull request closed without merging**